install.sh

```diff
@@ -287,9 +328,23 @@ if [ ! -f "$RELAY_CREDENTIALS_JSON" ]; then
   echo "Relay credentials written to $RELAY_CREDENTIALS_JSON"
 fi
 
-echo ""
-echo "----------------"
-echo "You're all done! Run the following command to get Sentry running:"
-echo ""
-echo "  docker-compose up -d"
-echo ""
+if [[ "$MINIMIZE_DOWNTIME" ]]; then
+  # Start the whole setup, except nginx and relay.
+  $dc up -d --remove-orphans $($dc config --services |grep -v -e '^nginx$' -e '^relay$')
+  $dc exec nginx service nginx reload
+
+  echo "Waiting for Sentry to start..."
+  while [[ "$(curl -m 1 http://localhost:9000/_health/ --silent)" != "ok" ]]; do
```

Comment:
Can there be a better way to do this for those of us not exposing port 9000?

I'm using Traefik as an edge router, so I don't expose any ports from Sentry at all.

My suggestion would be to keep `http://localhost:9000` as the default but allow overriding it, either with an environment variable (preferred) or an argument to the script. If using an argument, maybe it could be `--minimize-downtime` for no change to the URL and `--minimize-downtime=https://sentry.example.com` to override it.

Reply:
If you want, I can wait until this is merged and then open a new PR to add the behavior I need.

Reply:
Related: #512

Reply:
Good catch! I don't really care for the proxy here, so it should probably run in a container and wait for `web:9000`.

Reply:
I think we can merge #512 before to make things easier for this PR, that said we can also do something like https://unix.stackexchange.com/a/45943/108960 replacing the `tail -f` part with `$dc logs -f web` and look for some uWSGI/Django success message and avoid curling and ports altogether.

Reply:
I've changed it to waiting for `http://web:9000/_health/` from an alpine container - this way I don't have to think about the port, as I'm running inside the compose project's network. Plus this sheds the `curl` dependency on host :-)